tidb-lightning/tidb-lightning-distributed-import.md

```diff
@@ -89,9 +97,14 @@ status-addr = ":8289"
 data-source-dir = "/path/to/source-dir"
 
 [tikv-importer]
+<<<<<<< HEAD
 # Whether to allow importing data to tables with data. The default value is `false`.
 # When you use parallel import mode, you must set it to `true`, because multiple TiDB Lightning instances are importing the same table at the same time.
 # NOTE: this config field is introduced since v5.3.1. Ignore it when using TiDB Lightning v5.3.0.
+=======
+# Whether to allow importing data into tables that already have data. The default value is `false`.
+# When using parallel import, because multiple TiDB Lightning instances import a table at the same time, this configuration item must be set to `true`.
+>>>>>>> fe05a21fe (lightning: update distributed import (#13483))
```

Comment:
```suggestion
# Whether to allow importing data into tables that already have data. The default value is `false`.
# When using parallel import, because multiple TiDB Lightning instances import a table at the same time, this configuration item must be set to `true`.
# NOTE: this config field is introduced since v5.3.1. Ignore it when using TiDB Lightning v5.3.0.
```